src/OpenLoco/src/GameCommands/RemoveTree.cpp

```diff
@@ -1,27 +1,57 @@
+#include "Audio/Audio.h"
 #include "Economy/Economy.h"
 #include "Economy/Expenditures.h"
 #include "GameCommands.h"
+#include "Map/TileElement.h"
 #include "Map/TileManager.h"
 #include "Map/TreeElement.h"
 #include "Objects/ObjectManager.h"
 #include "Objects/TreeObject.h"
 #include "OpenLoco.h"
 #include "S5/S5.h"
+#include "TownManager.h"
+#include "ViewportManager.h"
 
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::GameCommands
 {
+    static loco_global<Core::Prng, 0x00525E20> _prng;
```

Comment:
In a future refactor we really need to cut down accessing this via the loco_global.